examples/README.md

```diff
@@ -9,12 +9,25 @@ This collection of sample templates will help you get started with MongoDB Atlas
 You must [configure API keys](https://www.mongodb.com/docs/atlas/configure-api-access/#std-label-atlas-admin-api-access) to authenticate with your MongoDB Atlas organization.
 
 ### AWS
-#### Activate the MongoDB Atlas CloudFormation public extensions
+### Activate the MongoDB Atlas CloudFormation public extensions
+**Option 1: Manually create IAM execution role and activate resources from AWS Console** 
+
 To activate a public extension, create an execution role and pass the ARN of the role as an input. Use [this template](execution-role.yaml) to create a [new CloudFormation stack](https://console.aws.amazon.com/cloudformation/home#/stacks/create) to create the execution role.
 
 You must then activate the Public extension from your AWS console. You have to do this in each AWS Account and in each AWS Region. Use [this link](https://us-east-1.console.aws.amazon.com/cloudformation/home#/registry/public-extensions?visibility=PUBLIC&type=RESOURCE&category=AWS_TYPES) to register extensions on CloudFormation.
 
-#### CloudFormation Profile
+**Option 2: Use provided CFN template to create IAM execution role and activate extensions**
+
+Use the [CFN template provided](activate-mongodb-atlas-resources.template.yaml) to create a new stack in required AWS region which will activate all required MongoDB Atlas extensions in your account in the specified region.
+
+_Note:_
+
+_1. All the `TypeActivation` resources are commented out by default in the template. Please use this template to activate only the required extensions by **_uncommenting_** otherwise you might get `"Rate exceeded (Service: CloudFormation, Status Code: 400)` error in your stack if all extensions are tried to be activated at once._
```

Comment:
To fix this: Have you tried to add `dependsOn` on each resource? You can make each resource DepensOn the one above to have a sequencing run. See [cloudformation-rate-exceeded-error](https://repost.aws/knowledge-center/cloudformation-rate-exceeded-error)

> Use the DependsOn attribute
> Unless a dependency is defined between resources, CloudFormation creates and updates resources at the same time. The [DependsOn](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-attribute-dependson.html) attribute defines dependencies between resources to control concurrent updates.

Reply:
Haven't tried that yet. Will give that try .

Reply:
Works with dependsOn, updated